src/components/CriteriaProvider.tsx

```diff
@@ -61,13 +70,13 @@ function elmLibraryToCriteria(elm: ElmLibrary, custom = false): Criteria[] {
   });
 }
 
-function jsonToCriteria(rawElm: string): Criteria[] | undefined {
-  const elm = JSON.parse(rawElm);
-  if (!elm.library?.identifier) {
+function jsonToCriteria(rawCriteria: string): Criteria[] | undefined {
+  const criteria = JSON.parse(rawCriteria);
+  if (!criteria.library?.identifier) {
     alert('Please upload ELM file');
     return;
   }
-  return elmLibraryToCriteria(elm);
+  return elmLibraryToCriteria(criteria);
 }
```

Comment:
So since this only has one branch now and the `elmLibraryToCriteria` requires the argument to be an `ElmLibrary` can we revert these changes back to the names `elm` instead of `criteria`?